tools/loris_db_upgrade.php

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/php
+<?php
+
+/**
+ * This script will upgrade the loris database to the lastest. 
+ * Usage: php loris_db_upgrade.php [-D]
+ *
+ * PHP Version 7
+ *
+ * @category Install
+ * @package  Loris
+ * @author   Shen Wang <shen.wang2@mcgill.ca>
+ * @license  Loris license
+ * @link     https://www.github.com/aces/Loris-Trunk/
```

Comment:
```suggestion
 * @link     https://www.github.com/aces/Loris/
```

remove old repo name